public/views/positionForm.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Position List</title>
+    <link rel="stylesheet" href="../styles/reset.css">
+    <link rel="stylesheet" href="../styles/styles.css">
+    <script src="../js/index.js"></script>
+    <script src="../js/positionForm.js"></script>
+    <script src="../js/positionForm.js"></script>
+</head>
+<body>
+    <header>
+        <a id="mindSetLogo" href="/"><span>Mind</span>SET</a>
+        <nav>
+          <a href="#">admins</a>
+          <a href="#">applications</a>
+          <a href="#">clients</a>
+          <a href="#">interviews</a>
+          <a id="positionsNav" href="../views/positionList.html">positions</a>
+          <a href="#">profiles</a>
+          <a href="#">psychologists</a>
+          <a href="#">sessions</a>
+        </nav>
+      </header>
+      <div class="container">
+        <h2 class="header">Position form</h2>
+        <form id="form">
+            <span>Client:</span>
+            <input id="client" placeholder="Client" required>
```

Comment:
the client input should be a `select` with the client's list. Please see how it was made [here](https://github.com/BaSD-2021/Esteban-MindSet-server/pull/45/files#diff-004136567525c4bbfdbb7f3cbb384fb080ae3150f3e5694eb5a247ee8754a89fR32)

Reply:
the same for the profiles input

Reply:
done!

Reply:
In profiles, was given me lots of trouble to populate, so i leave it for last, if i have time

Reply:
done profiles also